src/views/Home.js

```diff
@@ -28,33 +36,56 @@ class Home extends React.Component {
 
   componentDidMount() {
     Home.fetchData(this.props.dispatch);
+    window.addEventListener('resize', this.handleResize);
   }
 
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleResize);
+  }
+
+  handleResize() {
+    this.setState({ isMobile: window.innerWidth < 768 });
+  }
+
+
   render() {
     const {formatMessage} = this.props.intl;
     const {topHearing, openHearings, language} = this.props;
-    return (<div className="container">
-      <Helmet title={formatMessage({id: 'welcome'})}/>
-      <h1><FormattedMessage id="welcome"/></h1>
-      <p className="lead"><FormattedMessage id="welcomeMessage"/></p>
-      {topHearing && <FullWidthHearing hearing={topHearing}/>}
-      <hr />
-      <Row>
-        {openHearings && !openHearings.isFetching &&
+    const {isMobile} = this.state;
+    const hearingMap = (openHearings && openHearings.data ? (<div className="map">
```

Comment:
Add the same 15px left and right padding for the hearing map element as in the rest of the page elements.